settings.gradle.kts

```diff
@@ -36,6 +37,7 @@ include(
     "libs:etterlatte-jobs",
     "libs:etterlatte-kafka",
     "libs:etterlatte-pdl-model",
+    "libs:etterlatte-institusjonsopphold",
```

Comment:
Burde vi kalt dette biblioteket  `etterlatte-institusjonsopphold-model` for å være konsekvent og tydelig på hva det inneholder?

Reply:
Ja det kan jeg være enig i. Fikser.

Reply:
865523ef144ae71f1ff7e955786c2750e4f90894